packages/app-contracts/src/RemoveABI.tsx

```diff
@@ -62,33 +46,16 @@ class RemoveABI extends React.PureComponent<Props> {
           <Button.Or />
           <Button
             isPrimary
-            onClick={this.onRemove}
+            onClick={(): void => {
```

Comment:
Can we split this, i.e. `const _onClick = (): void => {...}` - still not perfect since it will still be created each render, but better.

Reply:
Or probably `_onRemove`